src/Form/ExerciseType.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\Form;
+
+use App\Entity\Exercise;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Bridge\Doctrine\Form\Type\EntityType;
+use App\Entity\MuscleGroup;
+
+class ExerciseType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add('name')
+            ->add('muscleGroups', EntityType::class, [
+                'class' => MuscleGroup::class,
+                'choice_label' => 'name',
+                'multiple' => true,
+                'expanded' => true,
+                'group_by' => fn(MuscleGroup $muscle) => $muscle->getCategory() ? $muscle->getCategory()->getName() : 'Uncategoriized',
```

Comment:
Відповідно до бест-пректіс по стилізації коду - прийнято уникати рядків, які перевищують довжину в 120 символів.
```suggestion
                'group_by' => 
                    fn(MuscleGroup $muscle) => $muscle->getCategory() 
                        ? $muscle->getCategory()->getName() 
                        : 'Uncategoriized',
```